src/components/projects/jguide/tabs/GettingHigh.astro

```diff
@@ -0,0 +1,59 @@
+---
+import gettingHigh from "@project-images/jguide/getting-high.png"
+import TabDetails from "./TabDetails.astro";
+---
+<p class="intro-text">
+    “Getting High”, is about the biology of how we get high. 
+    <br/>
+    <br/>
```

Comment:
Don't know what this powers, but this is not the best approach. I'm curious to know why you thought to do this.

Reply:
Hey
The design has all the text together so it's not particularly easy to get the exact spacing
But I can see from the text that there are two line breaks so this is just a natural way to implement those line breaks instead of trying to use margin to get exact spacing